src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -536,15 +537,16 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
-  const renderLiveScheduleToggle = () => (
+  const renderShowScheduleToggle = () => (
```

Comment:
Should we `useMemo` this? It doesn't look like a very expensive component but still...

Reply:
No real benefit. It’s a function that will always return the same JSX, so memoing it will have do nothing different. Because the function is called to produce its output rather than being passed as a handler, it doesn’t matter that it’s not a stable function reference.

it’s tech debt for sure, and a pattern we want to eliminate from the codebase. But this being admin v1 legacy, not worth spending the time to refactor.